cypress/e2e/upload-download.cy.ts

```diff
@@ -0,0 +1,21 @@
+import {UploadPage} from "../page/index";
+
+describe("Test to upload and download a file", () => {
+  let uploadPage: UploadPage;
+  const fileName = "prueba.txt";
+
+  beforeEach(() => {
+    uploadPage = new UploadPage();
+  });
+
+  it("Upload file", () => {
+    // Arrange
+    uploadPage.visitUploadPage();
+
+    // Action
+    uploadPage.uploadFile(fileName);
+
+    // Assert
+    uploadPage.getTitle(fileName);
```

Comment:
could you change the name of this method to be `checkFileName` or `verifyFileName`

Reply:
Yes sure